pkgs/tools/misc/iay/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, nixosTests
+, openssl
+, pkg-config
+, Security
+, Foundation
+, Cocoa
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "iay";
+  version = "0.4.0";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
```

Comment:
`src` should be here, instead of `buildInputs`

Reply:
Done